examples/animation/unchained.py

```diff
@@ -25,7 +25,7 @@
 # Generate random data
 data = np.random.uniform(0, 1, (64, 75))
 X = np.linspace(-1, 1, data.shape[-1])
-G = 1.5 * np.exp(-4 * X * X)
```

Comment:
In most cases I think this construct is marginally faster.

Reply:
This is not a useful measurement, because numpy hard-codes `power(x, 2)` to fall back on `square(x)`.